### PR TITLE
Giving credit to Ethan White's support

### PR DESCRIPTION
**Pull request by gvwilson**
@ethanwhite @pbanaszkiewicz please review

### CODE REVIEW

**Comment by ethanwhite** [comment]
This is fine, but it also doesn't need to go on every page if you don't want (e.g., it could just be at the bottom of the README or in an acknowledgements file. I'm happy with whatever.

File: workshops/templates/base_without_navbar.html [hunk 1]
```diff
@@ -49,6 +49,8 @@
             (<a href="https://github.com/swcarpentry/amy/commit/{{workshops_git_hash}}">{{workshops_git_short_hash}}</a>
             {{workshops_git_date}}{% if workshops_git_dirty %} dirty{% endif %})
           {% endif %}
+	  <br/>
+	  This project was funded in part by a grant from the National Science Foundation (0953694) to Prof. Ethan White.
```
Comment (pbanaszkiewicz) [inline workshops/templates/base_without_navbar.html]: I'd link to Ethan's website, but it's not working? http://whitelab.weecology.org/ cc @ethanwhite

File: workshops/templates/base_without_navbar.html [hunk 2]
```diff
@@ -49,6 +49,8 @@
             (<a href="https://github.com/swcarpentry/amy/commit/{{workshops_git_hash}}">{{workshops_git_short_hash}}</a>
             {{workshops_git_date}}{% if workshops_git_dirty %} dirty{% endif %})
           {% endif %}
+	  <br/>
```
Comment (pbanaszkiewicz) [inline workshops/templates/base_without_navbar.html]: Both "Powered by AMY version 1.1.0-dev (376426b 2015-10-23)" and "This project was funded in part by a grant from the National Science Foundation (0953694) to Prof. Ethan White." should be in separate paragraphs (`<p>`).

**Comment by ethanwhite** [comment]
> I'd link to Ethan's website, but it's not working? http://whitelab.weecology.org/ cc @ethanwhite

Weird, thanks for the heads up. I just kicked it and it looks like everything is working again.

**Pull request merged**